src/controllers/typings/demoRequest.controller.d.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * All all entry-point typings for request body of controller using their validators here
+ */
+
+import * as demoRequestControllerValidator from '@/controllers/validators/demoRequest.controller.validation';
+import { z } from 'zod';
+
+/**
+ * naming convention - {controller_method_name_in_CapitalizeCase}${RequestBody/RequestParams/RequestQuery}
```

Comment:
controller_method_name_in_CapitalizeCase 
MAKE IT 
controller_method_name_in_camelCase

Reply:
Done